src/utils/lzConfigManager.ts

```diff
@@ -126,6 +128,63 @@ export class LayerZeroConfigManager {
         return contracts;
     }
 
+    public createContract(contractName: string) {
+        const config = this.getConfigObject();
+        const contracts = this.getArrayProperty(config, "contracts");
+
+        if (contracts && !contracts.getText().includes(contractName)) {
+            contracts.addElement(`{ contract: ${contractName} }`);
+        }
+        this.sourceFile.saveSync();
+    }
+
+    public cacheOmniPointHardhatObjects() {
+        // get variable declarations
+        const variablesWithOmniPointHardhat = this.sourceFile.getVariableDeclarations().filter(variable => {
+            const typeNode = variable.getTypeNode();
+            return typeNode && typeNode.getText() === "OmniPointHardhat";
+        });
+        return variablesWithOmniPointHardhat.map(variable => {
+            const key = variable.getName();
+            const value = variable.getInitializer()?.getText();
+            return { [key]: value };
+        }).reduce((acc, curr) => ({ ...acc, ...curr }), {});
+    }
+
+    public isOmniPointHardhatObject(name: string) {
+        return Object.keys(this.oAppOmniGraphHardhat).includes(name);
+    }
+
+    public createOmniPointHardhatObject(name: string, objectLiteralText: string) {
+        // Find the target node where the new variable will be inserted before
+        const configDeclaration = this.sourceFile.getVariableDeclaration("config");
+        if (configDeclaration) {
+            const configStatement = configDeclaration.getVariableStatement();
+            if (configStatement) {
+                const insertPosition = configStatement.getChildIndex();
+
+                // Insert the new variable statement right before the config statement
+                this.sourceFile.insertStatements(insertPosition, [
+                    `const ${name}: OmniPointHardhat = ${objectLiteralText};`
+                ]);
+
+                // Save the updated source file
+                this.sourceFile.saveSync();
+
+                // Update cached OmniPointHardhat objects
+                this.oAppOmniGraphHardhat = this.cacheOmniPointHardhatObjects();
+            }
+        }
+    }
+
+    public deleteOmniPointHardhatObject(name: string) {
+        const variableDeclaration = this.sourceFile.getVariableDeclaration(name);
+        if (variableDeclaration) {
+            variableDeclaration.remove();
+            this.sourceFile.saveSync();
```

Comment:
is this an async or sync call and if so any need to do any error logging if the saveSync doesnt go well?